src/encoded/static/components/home.js

```diff
@@ -710,114 +712,203 @@ ChartGallery.defaultProps = {
 };
 
 
-// Component to allow clicking boxes on classic image
-class AssayClicking extends React.Component {
-    constructor(props) {
-        super(props);
+// Defines characteristics of the svg rectangles overlaid on the classic image.
+//   queryValue: Used in queries to the server to filter assays.
+//   voice: Text for screen readers to say.
+const assayList = [
+    {
+        queryValue: '3D chromatin structure',
```

Comment:
objects are a collection of key/value pairs. If you key has the word "value" in it, it may be confusing to people who probably spend too much time thinking about theory. Plus, "query" is very generic. Would a better key be "Assayfilter"?